### PR TITLE
docs: nav updates for data sources

### PR DESCRIPTION
**Pull request by imatwawana**
Updates navigation-related text and images in Data Sources config topics.

### CODE REVIEW

File: docs/sources/datasources/azure-monitor/_index.md [hunk 1]
```diff
@@ -40,9 +40,12 @@ The Azure Monitor data source supports visualizing data from three Azure service
 
 **To access the data source configuration page:**
 
-1. Hover the cursor over the **Configuration** (gear) icon.
-1. Select **Data Sources**.
-1. Select the Azure Monitor data source.
+1. Click **Connections** in the left-side menu.
+1. Under Your connections, click **Data sources**.
+1. Enter `Azure Monitor` in the search bar.
+1. Select **Azure Monitor**.
```
Comment (chri2547) [inline docs/sources/datasources/azure-monitor/_index.md:46]: Is it a select or a click?

File: docs/sources/datasources/elasticsearch/_index.md [hunk 2]
```diff
@@ -35,22 +35,25 @@ This data source supports these versions of Elasticsearch:
...
-**To access the data source configuration page:**
+To configure basic settings for the data source, complete the following steps:
 
-1. Hover the cursor over the **Configuration** (gear) icon.
-1. Select **Data Sources**.
-1. Select the Elasticsearch data source.
+1. Click **Connections** in the left-side menu.
+1. Under Your connections, click **Data sources**.
+1. Enter `Elasticsearch` in the search bar.
+1. Select **Elasticsearch**.
```
Comment (chri2547) [inline docs/sources/datasources/elasticsearch/_index.md:43]: click vs select?

File: docs/sources/datasources/elasticsearch/_index.md [hunk 3]
```diff
@@ -35,22 +35,25 @@ This data source supports these versions of Elasticsearch:
...
-Set the data source's basic configuration options carefully:
+   The **Settings** tab of the data source is displayed.
 
-| Name        | Description                                                                |
-| ----------- | -------------------------------------------------------------------------- |
-| **Name**    | Sets the name you use to refer to the data source in panels and queries.   |
-| **Default** | Sets the data source that's pre-selected for new panels.                   |
-| **Url**     | Sets the HTTP protocol, IP, and port of your Elasticsearch server.         |
-| **Access**  | Don't modify Access. Use "Server (default)" or the data source won't work. |
+1. Set the data source's basic configuration options carefully:
```
Comment (chri2547) [inline docs/sources/datasources/elasticsearch/_index.md:47]:
```suggestion
1. Set the data source's basic configuration options:
```

File: docs/sources/datasources/elasticsearch/_index.md [hunk 4]
```diff
@@ -35,22 +35,25 @@ This data source supports these versions of Elasticsearch:
...
 
-You must also configure settings specific to the Elasticsearch data source.
+   | Name        | Description                                                                |
+   | ----------- | -------------------------------------------------------------------------- |
+   | **Name**    | Sets the name you use to refer to the data source in panels and queries.   |
+   | **Default** | Sets the data source that's pre-selected for new panels.                   |
+   | **Url**     | Sets the HTTP protocol, IP, and port of your Elasticsearch server.         |
+   | **Access**  | Don't modify Access. Use "Server (default)" or the data source won't work. |
```
Comment (chri2547) [inline docs/sources/datasources/elasticsearch/_index.md:54]:
```suggestion
   | **Access**  | Don't modify Access. Use `Server (default)` or the data source won't work. |
```

File: docs/sources/datasources/google-cloud-monitoring/_index.md [hunk 5]
```diff
@@ -33,18 +33,21 @@ Once you've added the Google Cloud Monitoring data source, you can [configure it
...
-**To access the data source configuration page:**
+To configure basic settings for the data source, complete the following steps:
 
-1. Hover the cursor over the **Configuration** (gear) icon.
-1. Select **Data Sources**.
-1. Select the **Google Cloud Monitoring** data source.
+1. Click **Connections** in the left-side menu.
+1. Under Your connections, click **Data sources**.
+1. Enter `Google Cloud Monitoring` in the search bar.
+1. Select **Google Cloud Monitoring**.
```
Comment (chri2547) [inline docs/sources/datasources/google-cloud-monitoring/_index.md:41]: click vs select?

File: docs/sources/datasources/google-cloud-monitoring/_index.md [hunk 6]
```diff
@@ -33,18 +33,21 @@ Once you've added the Google Cloud Monitoring data source, you can [configure it
...
 
-Set the data source's basic configuration options carefully:
+   The **Settings** tab of the data source is displayed.
 
-| Name        | Description                                                              |
-| ----------- | ------------------------------------------------------------------------ |
-| **Name**    | Sets the name you use to refer to the data source in panels and queries. |
-| **Default** | Sets whether the data source is pre-selected for new panels.             |
+1. Set the data source's basic configuration options carefully:
```
Comment (chri2547) [inline docs/sources/datasources/google-cloud-monitoring/_index.md:45]:
```suggestion
1. Set the data source's basic configuration options:
```

File: docs/sources/datasources/jaeger/_index.md [hunk 7]
```diff
@@ -27,22 +27,25 @@ You can also [upload a JSON trace file]({{< relref "#upload-a-json-trace-file" >
...
+To configure basic settings for the data source, complete the following steps:
+
+1. Click **Connections** in the left-side menu.
+1. Under Your connections, click **Data sources**.
+1. Enter `Jaeger` in the search bar.
+1. Select **Jaeger**.
+
+   The **Settings** tab of the data source is displayed.
+
+1. Set the data source's basic configuration options carefully:
```
Comment (chri2547) [inline docs/sources/datasources/jaeger/_index.md:39]:
```suggestion
1. Set the data source's basic configuration options:
```

File: docs/sources/datasources/loki/_index.md [hunk 8]
```diff
@@ -26,21 +26,24 @@ Once you've added the Loki data source, you can [configure it]({{< relref "#conf
...
-| **Name**            | Sets the name you use to refer to the data source in panels and queries.                                                                                            |
-| **Default**         | Sets the data source that's pre-selected for new panels.                                                                                                            |
-| **URL**             | Sets the HTTP protocol, IP, and port of your Loki instance, such as `http://localhost:3100`.                                                                        |
-| **Allowed cookies** | Defines which cookies are forwarded to the data source. Grafana Proxy deletes all other cookies.                                                                    |
-| **Maximum lines**   | Sets the upper limit for the number of log lines returned by Loki. Defaults to 1,000. Lower this limit if your browser is sluggish when displaying logs in Explore. |
+1. Set the data source's basic configuration options carefully:
```
Comment (chri2547) [inline docs/sources/datasources/loki/_index.md:38]:
```suggestion
1. Set the data source's basic configuration options:
```

File: docs/sources/datasources/opentsdb/_index.md [hunk 9]
```diff
@@ -24,17 +24,27 @@ Administrators can also [configure the data source via YAML]({{< relref "#provis
...
+To configure basic settings for the data source, complete the following steps:
+
+1.  Click **Connections** in the left-side menu.
+1.  Under Your connections, click **Data sources**.
+1.  Enter `OpenTSDB` in the search bar.
+1.  Select **OpenTSDB**.
+
+    The **Settings** tab of the data source is displayed.
+
+1.  Set the data source's basic configuration options carefully:
```
Comment (chri2547) [inline docs/sources/datasources/opentsdb/_index.md:36]:
```suggestion
1.  Set the data source's basic configuration options:
```

File: docs/sources/datasources/postgres/_index.md [hunk 10]
```diff
@@ -22,24 +22,34 @@ Administrators can also [configure the data source via YAML]({{< relref "#provis
...
+To configure basic settings for the data source, complete the following steps:
+
+1.  Click **Connections** in the left-side menu.
+1.  Under Your connections, click **Data sources**.
+1.  Enter `PostgreSQL` in the search bar.
+1.  Select **PostgreSQL**.
+
+    The **Settings** tab of the data source is displayed.
+
+1.  Set the data source's basic configuration options carefully:
```
Comment (chri2547) [inline docs/sources/datasources/postgres/_index.md:34]:
```suggestion
1.  Set the data source's basic configuration options:
```

File: docs/sources/datasources/prometheus/_index.md [hunk 11]
```diff
@@ -34,46 +34,51 @@ For more information on how to query other Prometheus-compatible projects from G
...
+To configure basic settings for the data source, complete the following steps:
+
+1.  Click **Connections** in the left-side menu.
+1.  Under Your connections, click **Data sources**.
+1.  Enter `Prometheus` in the search bar.
+1.  Select **Prometheus**.
+
+    The **Settings** tab of the data source is displayed.
+
+1.  Set the data source's basic configuration options carefully:
```
Comment (chri2547) [inline docs/sources/datasources/prometheus/_index.md:46]:
```suggestion
1.  Set the data source's basic configuration options:
```

File: docs/sources/datasources/tempo/_index.md [hunk 12]
```diff
@@ -27,22 +27,26 @@ You can also [use the Service Graph]({{< relref "#use-the-service-graph" >}}) to
...
-1. Select **Data Sources**.
-1. Select the Tempo data source.
+1.  Click **Connections** in the left-side menu.
+1.  Under Your connections, click **Data sources**.
+1.  Enter `Tempo` in the search bar.
+1.  Select **Tempo**.
+
+    The **Settings** tab of the data source is displayed.
+
+1.  Set the data source's basic configuration options carefully:
```
Comment (chri2547) [inline docs/sources/datasources/tempo/_index.md:39]:
```suggestion
1.  Set the data source's basic configuration options:
```

File: docs/sources/datasources/zipkin/_index.md [hunk 13]
```diff
@@ -27,22 +27,26 @@ You can also [upload a JSON trace file]({{< relref "#upload-a-json-trace-file" >
...
+To configure basic settings for the data source, complete the following steps:
+
+1.  Click **Connections** in the left-side menu.
+1.  Under Your connections, click **Data sources**.
+1.  Enter `Zipkin` in the search bar.
+1.  Select **Zipkin**.
+
+    The **Settings** tab of the data source is displayed.
+
+1.  Set the data source's basic configuration options carefully:
```
Comment (chri2547) [inline docs/sources/datasources/zipkin/_index.md:39]:
```suggestion
1.  Set the data source's basic configuration options:
```

**Review by chri2547 — APPROVED** [review]
a few nits

File: docs/sources/datasources/azure-monitor/_index.md (hunk 1 above)
Comment (imatwawana) [inline docs/sources/datasources/azure-monitor/_index.md:46]:
```suggestion
1. Click **Azure Monitor**.
```

File: docs/sources/datasources/elasticsearch/_index.md (hunk 2 above)
Comment (imatwawana) [inline docs/sources/datasources/elasticsearch/_index.md:43]:
```suggestion
1. Click **Elasticsearch**.
```

File: docs/sources/datasources/google-cloud-monitoring/_index.md (hunk 5 above)
Comment (imatwawana) [inline docs/sources/datasources/google-cloud-monitoring/_index.md:41]:
```suggestion
1. Click **Google Cloud Monitoring**.
```

**Pull request merged**